roles/imageset_mirroring/README.md

```
@@ -0,0 +1,205 @@
+## Requirements
+
+- `oc-mirror` binary (automatically downloaded by the role)
+- Network access to source and target registries
+- Valid authentication for registries (if required)
+
+## Role Variables
+
+### Common Variables
+
+| Variable                       | Type    | Required | Default | Description                                                                                          |
+| `im_task`                      | string  | yes*     | -       | Mirroring action to perform. Valid values: `latest`, `all`, `custom`.                                |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)